packages/plugins/rubygem-smart_proxy_salt/rubygem-smart_proxy_salt.spec

```diff
@@ -23,18 +23,20 @@
 %global gem_name smart_proxy_salt
 %global plugin_name salt
 
-%global foreman_proxy_min_version 1.25
+%global foreman_proxy_min_version 2.5
 %global foreman_proxy_dir %{_root_datadir}/foreman-proxy
 %global foreman_proxy_statedir %{_root_localstatedir}/lib/foreman-proxy
 %global foreman_proxy_bundlerd_dir %{foreman_proxy_dir}/bundler.d
 %global foreman_proxy_settingsd_dir %{_root_sysconfdir}/foreman-proxy/settings.d
-%global smart_proxy_dynflow_bundlerd_dir %{_datadir}/smart_proxy_dynflow_core/bundler.d
 
 %global salt_config_dir %{_root_sysconfdir}/salt
+%global salt_proxy_runners_dir %{foreman_proxy_dir}/salt/runners
+%global salt_proxy_reactors_dir %{foreman_proxy_dir}/salt/reactors
+%global salt_state_grains_dir %{foreman_proxy_statedir}/salt/grains
 
 Summary: SaltStack support for Foreman Smart-Proxy
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 3.1.2
+Version: 4.0.0
 Release: 9%{?foremandist}%{?dist}
```

Comment:
```suggestion
Release: 1%{?foremandist}%{?dist}
```

Reply:
Can you explain to me, why this change caused the error? Was it just incorrect to introduce the change in the first place?

Reply:
The logic is that version should always follow the source (in this case smart_proxy_salt) so you have no control over that. The release is there to track the "version" of the spec/package. This then results in `%{version}-%{release}`, here `4.0.0-1` vs `4.0.0-9`. This is taken into account when comparing. As you can see, increasing the release makes it newer even though the version stays the same. However, `4.0.0-1` is already newer than `3.1.2-9` and we don't want to keep increasing release. You would end with a value that's insanely high and doesn't say anything anymore. While technically it wouldn't matter (computers understand it just fine), it's harder for the humans. That's why we always reset the release to 1 when we bump the version.

Reply:
Alright, thanks for the explanation! I'm gonna squash the commits and then, we are ready to merge, I guess?